Synergism.js

```diff
@@ -1320,32 +1320,24 @@ function format(input, accuracy, long) {
             standard = Math.floor(standard);
         }
         // Turn the number to string
-        standard = standard.toString();
+        const standardString = standard.toString();
         // Split it on the decimal place
-        let split = standard.split('.');
-        // Get the front half of the number (pre-decimal point)
-        let front = split[0];
-        // Get the back half of the number (post-decimal point)
-        let back = split[1];
+        const [front, back] = standardString.split('.');
         // Apply a number group 3 comma regex to the front
-        front = front.replace(/(\d)(?=(\d{3})+$)/g, "$1,");
+        const frontFormatted = BigInt(front).toLocaleString();
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Browser_compatibility

Breaks compatibility with legacy Edge and versions of Safari from before last week's release of Safari 14, if those browsers matter? (idk)

Reply:
good catch, legacy Edge isn't a big deal but Safari could be a problem.